### DISCUSSION TITLE
Quarto document templates?

### DISCUSSION THREAD

**Discussion by aalexandersson** [quarto-dev/quarto-cli#983] [Q&A] [answered]
How can you create **Quarto document templates** for automated reporting, similar to [R Markdown document templates](https://bookdown.org/yihui/rmarkdown/document-templates.html)?

A related `rmarkdown` reference is [draft.R](https://pkgs.rstudio.com/rmarkdown/reference/draft.html) to create a new document based on a template. Does `draft.R` work with Quarto? If so, how? If not, what is the equivalent solution to `draft` in Quarto? 
 
A related but much more advanced discussion about templates is [#961](https://github.com/quarto-dev/quarto-cli/discussions/961). In contrast, I "just" want to create document templates in Quarto like in R Markdown but I want to use Quarto because of its' other new or improved features (such as parameters and cross-language compatibility).

**Comment by jjallaire** [comment, marked as answer]
We are working on this now and expect to release/document within the next few weeks

**Comment by aalexandersson** [comment]
I unmarked the answer because of [issue 1111](https://github.com/quarto-dev/quarto-cli/issues/1111) which moves the task from the 1.0 milestone due on July 1 to the Formats milestone without a due date. The two answers seem contradictory. Please clarify: Approximately when do you expect to release/document Quarto document templates?

**Reply by jjallaire** [reply]
It's still in v1.0 but we are tracking it in a separate milestone (now renamed to 1.0 Formats)
